lib/takeaway.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'message'
+
+class Takeaway
+  attr_reader :menu, :order
+
+  def initialize
+    @menu = Menu.new
+    @order = Order.new
+    @message = Message.new
```

Comment:
Could dependency injection here be used with the Menu, Order and Message objects.